tests/test_blockchain.py

```
@@ -62,6 +64,41 @@ def test_blockchain_verification_success(client, db_session):
+    assert response.status_code == 201
+    issue_id = response.json()["id"]
+
+    # Verify issue in DB has previous_integrity_hash
+    issue2 = db_session.query(Issue).filter(Issue.id == issue_id).first()
+    assert issue2.previous_integrity_hash == hash1
+
+    # Verify blockchain-verify endpoint uses the stored hash
+    # (Implicitly tested if it returns is_valid=True and we know we didn't mock the internal DB query in the router)
+    response = client.get(f"/api/issues/{issue_id}/blockchain-verify")
+    assert response.status_code == 200
+    assert response.json()["is_valid"] == True
```

Ruff (0.15.2)

[error] Avoid equality comparisons to `True`; use `response.json()["is_valid"]:` for truth checks

Replace with `response.json()["is_valid"]`

(E712)